### DISCUSSION TITLE
Eclipse Theia v1.56.0

### DISCUSSION THREAD

**Discussion by sgraband** [eclipse-theia/theia#14547] [Release Announcements] [answered]
Hey everyone 👋,

The Eclipse Theia v1.56.0 release is happening today, starting around 1 PM CET.
If you have any nearly-complete PRs that need to be included, let us know in this thread before we begin.

We’ll post updates when the release starts and once it’s done.
Please avoid merging any PRs during the release until we confirm it's over.

**Comment by JonasHelming** [comment]
Nearly completed, to be included: https://github.com/eclipse-theia/theia/pull/14535

**Reply by JonasHelming** [reply]
it has just been merged

**Comment by sgraband** [comment]
The release will start now, We’ll post an update once it has completed.

**Comment by sgraband** [comment, marked as answer]
The release has completed, thank you to everyone that participated and contributed!

**Reply by JonasHelming** [reply]
[News & Noteworthy](https://eclipsesource.com/blogs/2024/12/06/eclipse-theia-1-56-release-news-and-noteworthy/)
